packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +151,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+        }
+        .day,
+        .interact .day,
+        .week-day {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+            fill: currentcolor;
+            border-color: ${SystemColors.Canvas}
+        }
+        .interact .day:not(.disabled, .inactive):hover {
+            background: ${SystemColors.ButtonFace};
+            color: ${SystemColors.ButtonText};
+            outline: 1px solid ${SystemColors.Highlight};
+        }
+        .week-day:${focusVisible},
+        .interact .day:not(.disabled, .inactive):${focusVisible} {
+            forced-color-adjust: none;
+            background: ${SystemColors.ButtonFace};
+            border-color: transparent;
+            ${focusTreatmentBase}
+            outline-color: ${SystemColors.Highlight}
+        }
+        .day.disabled,
+        .day.inactive {
+            color: ${SystemColors.GrayText};
+            opacity:1;
+            outline: none;
+        }
+        .day.inactive:${focusVisible} {
+            outline: calc(${focusStrokeWidth} * 1px) solid ${SystemColors.GrayText};
+            box-shadow: none;
+        }
+        .day.selected,
+        .day.selected:not(.disabled, .inactive):hover {
+            color: ${SystemColors.Highlight};
+            outline: 1px solid ${SystemColors.Highlight}
+        }
+        .interact .today,
+        .today,
+        .today .date ,
+        .interact .today:not(.disabled, .inactive):hover {
+            forced-color-adjust: none;
+            background: ${SystemColors.Highlight};
+            color: ${SystemColors.HighlightText};
+            fill: currentcolor;
```

Comment:
Same as above.

Reply:
this is still needed for this style
<img width="57" alt="image" src="https://user-images.githubusercontent.com/37851220/199157740-db3d24bc-6baf-40ad-955e-3dc29842d762.png">

 if we remove this, then we will not see any item that is considered "today" will not looked selected.